app/bundles/CoreBundle/Loader/ParameterLoader.php

```diff
@@ -97,10 +97,10 @@ public static function getLocalConfigFile(string $root, bool $updateDefaultParam
 
         if (!isset($paths['local_config'])) {
             if ($updateDefaultParameters) {
-                self::$defaultParameters['local_config_path'] = $root.'/config/local.php';
+                self::$defaultParameters['local_config_path'] = $root.'../config/local.php';
             }
 
-            return $root.'/config/local.php';
+            return $root.'../config/local.php';
```

Comment:
Are you confident that the slash doesn't have to be added as it was previously starting with one? Like
```php
return $root.'/../config/local.php';
```
There are 3 occurrences in this file.